### PR TITLE
New Drink: The Himalayan 

### PR DESCRIPTION
**Pull request by Sneeker134**
## About The Pull Request

Adds a drink to the bar, the Himalayan, which rapidly freezes the drinker and has the potential to slowly rack up damage through "brain freezes." Made from ice, frost oil, vodka, and tonic water.

himalayan

## Why It's Good For The Game

I was going through reagents generated through botany that the bar doesn't have a good drink for yet, and stumbled across Frost Oil. You can make iced beer with it but that drink didn't seem to do anything particularly unique. So, I decided to add this drink! It rapidly cools you down, and has a small chance to give you a brain freeze, which inflicts some minor brain damage. Also does the ice watcher thing of freezing you solid. Adding more stuff for bartenders to make is always fun, and adding more reason for bartender to interact with other departments helps them do more besides sit behind their counter the whole shift.

My only real concern is the rate of the brain damage. I don't think this drink should really be viable as a poison, but if you drink a ridiculous amount of it I could see being able to justify it having some more lasting effects. I'm not really sure how much brain damage this realistically inflicts, so any feedback on that part would be welcome. The art also isn't great, but I'm not a pixel artist so...

## Changelog
:cl:
add: New drink, the Himalayan. Rapidly cools down whoever consumes it,  potentially giving them a brain freeze!
/:cl:

### CODE REVIEW

**Comment by Arathianen** [comment]
Name is lizardbane instead and we have a deal.

**Comment by evilcyborgmrt** [comment]
More drinks is ok. More drinks that actually do things is good.

**Comment by FantasticFwoosh** [comment]
Reading this i thought it would make you intake less O2 to breathe in thin air, but whatever this is neat too.

**Comment by wesoda25** [comment]
Impedrezene is a pretty weak brain toxin, and it has a 80% chance each tick of giving 2 brain damage. So this probably won't be _that_ bad.

Might wanna dial it back though so that people are more willing to drink it, and it becomes less of a grief drink.

**Comment by Sneeker134** [comment]
> Impedrezene is a pretty weak brain toxin, and it has a 80% chance each tick of giving 2 brain damage. So this probably won't be _that_ bad.
> 
> Might wanna dial it back though so that people are more willing to drink it, and it becomes less of a grief drink.

1/16 the power of a pretty weak brain toxin seems like it probably won't end up doing much, but it seems high enough that if you drank a gigantic amount then you might start running into some issues, which is where I personally think the sweet spot is out. Its not like there is much reason to make this except for the bounty anyway, though I suppose it could be useful in keeping your temperature low if you plan on doing something really stupid.

**Comment by Iatots** [comment]
Maybe the brain damage could be dependant on your temperature?

**Comment by Sneeker134** [comment]
> Maybe the brain damage could be dependent on your temperature?

Seems like an interesting idea and has potential meme dream potential with a cryo-sting bartender. I don't really like the idea that people using this to cool themselves down wouldn't really end up suffering much of the effects from it though either. I'll consider it, but I kind of like it affecting everyone evenly, since lizard men are already going to get boned with the temperature drop anyway; double dipping on the brain damage seems kind of cruel.

**Comment by fludd12** [comment]
Make its' overdose be removing all of itself from your blood and freezing you in a solid block of ice like icewatchers can.

**Comment by Sneeker134** [comment]
> Make its' overdose be removing all of itself from your blood and freezing you in a solid block of ice like icewatchers can.

I didn't even know that _was_ a status effect! I think having everything tied together is pretty nice, so instead of an overdose effect I'm just gonna tie it up with the brain freeze. I'll try and add it in and test it out. Thanks for the idea!

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2178,18 +2178,19 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
 
 /datum/reagent/consumable/ethanol/himalayan/on_mob_life(mob/living/carbon/M)
-	// Cools the user down while the reagent is in the body. Occasionally causes a brain freeze, giving you a case of minor brain damage.
+	// Cools the user down while the reagent is in the body. Occasionally causes a brain freeze, giving you a case of minor brain damage and freezing you solid.
 	M.adjust_bodytemperature(-25 * TEMPERATURE_DAMAGE_COEFFICIENT)
 	if (prob(5))
 		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2)
-		to_chat(M, "<span class='notice'>["Your head pounds; you have a brain freeze!"]</span>")
+		M.apply_status_effect(/datum/status_effect/freon/watcher)
+		to_chat(M, "<span class='notice'>["Your blood is so cold, it freezes the air around you! Even worse, you have a brain freeze!"]/span>")
```
Comment (fludd12) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
		to_chat(M, "<span class='notice'>"Your blood is so cold, it freezes the air around you! Even worse, you have a brain freeze!"</span>")
```
You don't need the square brackets (I think?) but notably, you were missing the opening triangle bracket to the closing span tag.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -2173,3 +2173,24 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjust_fire_stacks(1)
 		M.IgniteMob()
 	..()
+
+/datum/reagent/consumable/ethanol/himalayan
+	name = "Himalayan"
+	description = "Freezing cold; its a miracle"
+	boozepwr = 40
+	color = "##0000ff"
```
Comment (XDTM) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: one # too many, although it could still be functional

Comment (Sneeker134) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Must've not deleted the hastag when copy pasting the new color in, and I don't think how it looked in a syringe or beaker is something I tested in game. Oops!

**Comment by Sneeker134** [comment]
I'm too small brain to merge, consider the PR dead. Sorry. :(

**Pull request closed without merging**